source/puppet/5.5/lang_data_abstract.md

```diff
@@ -117,6 +117,72 @@ Note that you can use capture groups in the regular expressions, but they won't
 * `Pattern[/\A[a-z].*/]` --- matches any string that begins with a lowercase letter.
 * `Pattern[/\A[a-z].*/, /\Anone\Z/]` --- matches the above **or** the exact string `"none"`.
 
+### `Timespan`
+
+The `Timespan` data type matches a Timespan; [strings][], or [numbers][] that represent a time and date; or `default`, which represents an infinite point in the past or future.
```

Comment:
```suggestion
The `Timespan` data type matches Timespan values within a given range; specified with [strings][] or [numbers][] that represent duration; or `default`, which represents a positive or infinite duration.
```

Reply:
typo, corrected - should have read "an infinite negative or positive duration" - in practical terms "+ or - the largest possible (i.e. infinite) duration of time", as in "10 minutes or longer" (the upper end is positive infinity), or "you can subtract as much time as you like" (the lower end is negative infinity).